src/Voter/RequestContentIdentityVoter.php

```diff
@@ -40,32 +46,54 @@ class RequestContentIdentityVoter implements VoterInterface
      * @param string $requestKey The key to look up the content in the request
      *                           attributes
      */
-    public function __construct($requestKey)
+    public function __construct($requestKey, RequestStack $requestStack = null)
```

Comment:
i think you miss passing the stack to the local variable.

Reply:
btw: do we have  a teste then?

Reply:
the test is testing the deprecated way of doing this. i will add more tests.

Reply:
and fixed assigning the stack